### ISSUE TITLE
Percent Correct

### ISSUE THREAD

**Issue by lizrose17** [memphis-iis/mofacts-ies#1127]
Package Countries Tutor1.zip uploaded. The percent correct was over 100% for several items; as I got items incorrect, the percent correct decreased. 
![Screenshot (31)](https://user-images.githubusercontent.com/88344486/234707898-5307f192-d45e-4410-9b96-0d14c0196968.png)

**Comment by lizrose17** [comment]
Similarly, I noticed this when I practiced the Bridge Test. After I answered the first item, the second item had "Infinity%" as the percent correct. On the next item, the percentage was 1300%, and it continued decreasing from there.
![Screenshot (39)](https://user-images.githubusercontent.com/88344486/235507400-0611cacf-9be4-4855-9ae7-b563c5faa7b6.png)
![Screenshot (40)](https://user-images.githubusercontent.com/88344486/235507414-df11adb5-582d-42a5-a9f6-286c5f703ca3.png)
![Screenshot (41)](https://user-images.githubusercontent.com/88344486/235507431-4fddef8a-9d6c-42f6-a35e-3bdf9ac11bb8.png)
![Screenshot (42)](https://user-images.githubusercontent.com/88344486/235507440-f4593342-06b7-47dc-b7e7-79dc55b53df6.png)
